### ISSUE TITLE
Magento 2 :- Not able to save customer in admin side when street line Number was modified 

### ISSUE THREAD

**Issue by pradeeprcs** [magento/magento2#13766]
### Preconditions

1. M 2.1.7

### Steps to reproduce

1. Login to admin and got to Store->configuration->Customer->Customer Configuration-> Name and 
     Address Options
2. Set Number of Lines in a Street Address to 4 and save
3. Go Customer -> All Customer -> edit one of the customer
4.  Add a new address and set all 4 street lines with some values 
5. Save the Customer with 4 line street values
6.  Go back to Store->configuration->Customer->Customer Configuration-> Name and 
     Address Options
7.  Set Number of Lines in a Street Address to 3 and save
8.   Edit same customer which added a address with 4 street values 
9. with out doing any thing / modify the  address data where you can see only 3 line street just save the customer 
10. you can see error "Street Address" cannot contain more than 3 lines. 

### Expected result

1.  Should save the customer and address with only 3  line with out error 
2. in front end also only three line data should display
3. in fronend also customer should be saved

### Actual result

1. While save customer from admin  throw error ""Street Address" cannot contain more than 3 lines."

**Comment by magento-engcom-team** [comment]
@pradeeprcs, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by nadeefit6** [comment]
Issue cannot be reproduced in Magento Ver 2.2.1

**Comment by JancyAbraham** [comment]
I am working on this at #dmcdindia

**Comment by magento-engcom-team** [comment]
@JancyAbraham thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @pradeeprcs 
Looks like this issue is not reproducible with Magento 2.2 anymore. We are closing this ticket now.
Please reopen if the issue persists.

**Comment by ivankakurov** [comment]
This issue is not fixed. 
In magento 2.2.6 after change address line number in store, customer whit address whit more then lines can't edit in administration. 
After click button "Save and Continue Edit", 
magento return error:  "Street Address" cannot contain more than 2 lines." . 
Can't delete address in admin, can't add new adderss.  
The customer is total broke i can't change anything, because magento2 validate all customer addresses, every time when edit/delete customer.

*The big problem is that once you have the wrong address for this user, he can not add a product to the cart, can not login, can not delete the address.*

**Comment by onepack** [comment]
I have M2.3.0 installed. And now try to change an address in the backend after I changed the system setting that a customer can have 1 address line in the settings.
It cannot be done/ properly saved with the message from Magento that the address cannot contain more than 1 lines. "Street Address" cannot contain more than 1 lines.
After hitting the save it gets saved however and in the frontend renders as "Street 123 Street 123 Street 123". Throwing away the address!! and hitting save seems not to help as things cannot be properly saved. See attachment. The error is persistent.
![customers-magento-admin](https://user-images.githubusercontent.com/475598/50148199-2afd6b80-02b8-11e9-85b5-530f2529b269.png)

**Comment by onepack** [comment]
I changed the seting back to "system default" that is "2". After that I went back to the customer address and saw "3" address lines. So one more than the system value. All filled in with the same value.
There seems to be something wrong in the count of the lines.
At least I can save the address again now but this seems a bug.
![customers-magento-admin2](https://user-images.githubusercontent.com/475598/50148780-ac093280-02b9-11e9-946a-fe9d870edeb7.png)
